### PR TITLE
[bugfix] Fix language supporting setting in JiebaTokenizer

### PR DESCRIPTION
**Pull request by howl-anderson**
**Proposed changes**:
- rename `language_list` to `supported_language_list`

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @degiz will take a look at it as soon as possible ✨

**Review by degiz — APPROVED** [review]
Thanks a lot for fixing that 🚀

File: changelog/6804.bugfix.md [hunk 1]
```diff
@@ -0,0 +1 @@
+bugfix: rename `language_list` to `supported_language_list` for JiebaTokenizer
```
Comment (tmbo) [inline changelog/6804.bugfix.md:1]:
```suggestion
Rename `language_list` to `supported_language_list` for `JiebaTokenizer`.
```

**Comment by wochinge** [comment]
Please change the target to `master` or `2.1.x` once I release Rasa Open Source 2.1 this afternoon. 

@howl-anderson Seems like one test is currently failing

**Comment by howl-anderson** [comment]
@wochinge Sorry for the late reply, I will try to fix it today.

**Comment by howl-anderson** [comment]
@wochinge Please take a look!

**Review by wochinge — APPROVED** [review]

**Comment by wochinge** [comment]
Thanks for the fix 🚀

**Pull request merged**